### PR TITLE
fix: unit tests in windows (resolves file path and build issues in windows)

### PR DESCRIPTION
**Pull request by harsha509**
## Proposed changes
* Fix unit tests in windows.(Tests were failing due to path issues in windows)
* Deleted `packages/wdio-shared-store-service/src/@types/shared-store-service.d.ts` which is no longer used and blocks build in Windows

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by harsha509** [comment]
Screenshot 2022-03-11 at 1 52 59 PM
In MAC

**Comment by harsha509** [comment]
image
In Win

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -39,10 +39,10 @@ jobs:
         run: npm ci
       - name: Bootstrap Packages
         run: npm run setup-full
-      # Disable unit test as they started to fail in Windows only
-      # (https://github.com/webdriverio/webdriverio/issues/8090)
-      # - name: Lint / Dependencies / Unit / Smoke Tests
-      #   run: npm run test
+      Disable unit test as they started to fail in Windows only
+      (https://github.com/webdriverio/webdriverio/issues/8090)
```
Comment (christian-bromann) [inline .github/workflows/test.yml]:
```suggestion
```

Comment (harsha509) [reply .github/workflows/test.yml]: Removed with commit https://github.com/webdriverio/webdriverio/pull/8097/commits/42688bb6ba1d761d406d1864aa93842e87520ee4 !

**Review by christian-bromann — APPROVED** [review]
Thanks for the patch 👍 

I will go ahead and merge. I am not sure why our Mac tests are so flaky, will have to investigate it.

**Pull request merged**